packages/ios-client/Sources/FishjamClient/events/Event.swift

```diff
@@ -319,7 +319,6 @@ struct ConnectedEvent: ReceivableEvent, Codable {
 struct EndpointAddedEvent: ReceivableEvent, Codable {
     struct Data: Codable {
         let id: String
-        let type: EndpointType
```

Comment:
On iOS if server sends data with `type` and it isn't included here will it fail to decode and crash? As far as I remeber on iOS json decoder is strict about such things?

Reply:
It won't fail (I tested it). 
The default key decoding strategy ignores keys that are not present. 

It will fail if the server removes a key we require, but there is a catch block that will result in a message printing.